scout/commands/load/case.py

```diff
@@ -106,6 +106,13 @@ def case(
         LOG.debug("Stack trace: {}".format(traceback.format_exc()))
         raise click.Abort()
 
+    if config_data.get("human_genome_build") not in [37, 38]:
```

Comment:
👍

Reply:
And that is the point, it shouldn't be missing or different from 37 and 38 in the config files